### PR TITLE
Restructured the file a little

### PR DESCRIPTION
**Pull request by skitiz**
Added a `.gitignore` entry for `/env` and your shitty `/Comic` folder.
Update `run.py` with an option to leave the program if user enters `n`

### CODE REVIEW

**Comment by coder137** [comment]
Do not make changes to run.py (This is deprecated) Just use it as a reference
Use defRun.py it works perfectly

**Comment by skitiz** [comment]
Noted.

File: Python3/xkcdDownloader/FlaskApp/run.py [hunk 1]
```diff
@@ -3,6 +3,15 @@
...
+app = Flask(__name__)
+
+@app.route('/')
+def hello_name():
+    return render_template('index.html')
+
+if __name__ == '__main__':
+    app.run()
 """
 USE defRun.py, DO NOT USE THIS ANYMORE
```
Comment (skitiz) [inline Python3/xkcdDownloader/FlaskApp/run.py:16]: I just read this @coder137 , lol.

**Comment by skitiz** [comment]
Keep the pull request open, so I can keep adding commits to it.

**Comment by coder137** [comment]
Sure, When you are done adding everything ill merge it.

**Pull request merged**